app/assets/stylesheets/layout/_work_package_mobile.sass

```diff
@@ -93,7 +93,10 @@
             margin: 0.75rem 0 2.5rem 0
 
           .work-packages--panel-inner
-            padding-left: 0
+            padding-right: 0
```

Comment:
Why did you do this? Because of this the content sticks directly to screen border..
<img width="273" alt="bildschirmfoto 2018-06-29 um 15 14 15" src="https://user-images.githubusercontent.com/7457313/42094176-2ffeb99e-7baf-11e8-9c01-2c301797b25f.png">